bin/main.dart

```diff
@@ -2,8 +2,17 @@ import 'dart:collection';
 import 'dart:convert';
 import 'dart:io';
 import 'package:args/args.dart' show ArgParser;
+import 'package:http/http.dart' as http;
+import 'package:tuple/tuple.dart';
+
 import 'package:translator/translator.dart';
 
+// supported translation providers
```

Comment:
You can use /// as a documentation comment. You can check out how it is implemented in localized.dart file

These documentation comments are valuable for the pud.dev where the package is

Reply:
This is an internal method and shouldn't be documented. Still need to use \/\/\/ ?